drivers/d3d12/SCsub

```diff
@@ -138,6 +140,9 @@ else:
     extra_defines += [
         "HAVE_STRUCT_TIMESPEC",
     ]
+    if methods.using_gcc(env) and methods.get_compiler_version(env)["major"] < 13:
+        # `region` & `endregion` not recognized as valid pragmas.
+        env_d3d12_rdd.Append(CCFLAGS=["-Wno-unknown-pragmas"])
```

Comment:
Might need to add it to `env` too actually, as it doesn't seem to be enough. Some files from outside `drivers/d3d12` might be including DirectX headers directly or via `drivers/d3d12/` includes.